sycl/include/sycl/ext/oneapi/experimental/enqueue_functions.hpp

```diff
@@ -357,6 +288,75 @@ void nd_launch(queue Q, launch_config<nd_range<Dimensions>, Properties> Config,
   });
 }
 
+// Free function kernel enqueue functions
+template <auto *Func, typename... ArgsT>
+void single_task(queue Q, kernel_function_s<Func> KernelFunc, ArgsT &&...Args) {
+  (void)KernelFunc;
```

Comment:
```suggestion
```

same below.

Reply:
Thanks for the catch, the  functions that take `queue` actually use the parameter, it's the `handler` ones that don't.

Reply:
This has been taken care of.